extensions/amazon-lambda-resteasy/deployment/src/main/java/io/quarkus/amazon/lambda/resteasy/deployment/AmazonLambdaResteasyProcessor.java

```diff
@@ -1,23 +1,64 @@
 package io.quarkus.amazon.lambda.resteasy.deployment;
 
+import static java.util.Arrays.asList;
+
+import java.util.List;
 import java.util.Optional;
 
+import javax.ws.rs.Path;
+
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.ClassType;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.MethodInfo;
+import org.jboss.jandex.Type;
+
 import io.quarkus.amazon.lambda.resteasy.runtime.AmazonLambdaResteasyConfig;
 import io.quarkus.amazon.lambda.resteasy.runtime.AmazonLambdaResteasyRecorder;
+import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.annotations.ExecutionTime;
 import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.substrate.ReflectiveHierarchyBuildItem;
 import io.quarkus.resteasy.server.common.deployment.ResteasyInjectionReadyBuildItem;
 import io.quarkus.resteasy.server.common.deployment.ResteasyServerConfigBuildItem;
 
 public class AmazonLambdaResteasyProcessor {
+    private static final DotName PATH = DotName.createSimple(Path.class.getName());
+    private static final List<DotName> FILTER = asList(
+            DotName.createSimple(Object.class.getName()),
+            DotName.createSimple(String.class.getName()));
 
     AmazonLambdaResteasyConfig config;
 
+    @BuildStep
+    void discover(CombinedIndexBuildItem combinedIndexBuildItem,
+            BuildProducer<ReflectiveHierarchyBuildItem> reflectiveClasses) {
+        for (AnnotationInstance info : combinedIndexBuildItem.getIndex().getAnnotations(PATH)) {
```

Comment:
This does not look correct. This is a valid JAX-RS endpoint:

```
@Path 
public class Foo {

@GET 
public String hello() {
return "hello";
}
}
```

It is not the Path annotation that indicates that a method needs to be registered, it is the various method annotations. That said I am pretty sure this is already handled by io.quarkus.resteasy.server.common.deployment.ResteasyServerCommonProcessor#registerReflectionForSerialization

Reply:
This code was added solving a problem I thought I had but didn't.  I left it because I figured it'd be a jumping off point for the native image step but thinking more on it i think the existing code in the lambda module is all we need.  In fact, it's possible they will all condense in to one module anyway.  I'll remove it.